src/box/alter.cc

```diff
@@ -749,9 +758,20 @@ struct alter_space {
 	int n_rows;
 };
 
+/**
+ * Fiber cond that wakes up all waiters on every deletion of alter_space.
+ * Is created with the first alter_space object.
+ */
+struct fiber_cond *alter_space_delete_cond;
```

Comment:
Should be `static`. Also, better initialize it at compile-time rather than on-demand:

```C
static FIBER_COND(alter_space_delete_cond);
```

Reply:
Oh, thanks, I missed the compile time initializer somehow - fixed. Also, added `static` qualifier.